xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -117,28 +117,32 @@ def collectBlockArguments(number: int, block: Block):
 @dataclass
 class ReturnOpToMemref(RewritePattern):
 
-    return_target: dict[ReturnOp, CastOp | memref.Subview]
+    return_target: dict[ReturnOp, list[CastOp | memref.Subview]]
 
     @op_type_rewrite_pattern
     def match_and_rewrite(self, op: ReturnOp, rewriter: PatternRewriter, /):
+        store_list: list[memref.Store] = []
 
         parallel = op.parent_op()
         assert isinstance(parallel, scf.ParallelOp | gpu.LaunchOp | scf.For)
 
-        subview = self.return_target[op]
+        for j in range(len(op.arg)):
 
-        assert isinstance(subview, memref.Subview)
-        assert isa(subview.result.typ, MemRefType[Attribute])
+            subview = self.return_target[op][j]
 
-        assert (block := op.parent_block()) is not None
+            assert isinstance(subview, memref.Subview)
+            assert isa(subview.result.typ, MemRefType[Attribute])
+
+            assert (block := op.parent_block()) is not None
 
-        dims = len(subview.result.typ.shape.data)
+            dims = len(subview.result.typ.shape.data)
 
-        args = collectBlockArguments(dims, block)
+            args = collectBlockArguments(dims, block)
 
-        store = memref.Store.get(op.arg, subview.result, args)
+            store_list.append(memref.Store.get(op.arg[j], subview.result,
+                                               args))
 
-        rewriter.replace_matched_op([store])
+        rewriter.replace_matched_op([*store_list])
```

Comment:
```suggestion
        rewriter.replace_matched_op(store_list)
```

Reply:
I get this pylance type issue if I change it like that 
```
Argument of type "list[Store]" cannot be assigned to parameter "new_ops" of type "Operation | list[Operation]" in function "replace_matched_op"
  Type "list[Store]" cannot be assigned to type "Operation | list[Operation]"
    "list[Store]" is incompatible with "Operation"
    TypeVar "_T@list" is invariant
      "Store" is incompatible with "Operation"Pylance[reportGeneralTypeIssues](https://github.com/microsoft/pylance-release/blob/main/DIAGNOSTIC_SEVERITY_RULES.md#diagnostic-severity-rules)
```

Reply:
Oh, I see. Let's stick with that for now then.